docs/ref/reference.rst

```diff
@@ -1,3 +1,5 @@
+.. _reference:
```

Comment:
maybe refer to `how to` for beginners at the  start of the reference?

Reply:
Added.